.openshift-ci/post_tests.py

```diff
@@ -204,6 +209,16 @@ def check_stackrox_logs(self):
             timeout=PostTestsConstants.CHECK_TIMEOUT,
         )
 
+    def add_test_result(self, test_results):
+        print("Storing operator test results in JUnit format")
```

Comment:
This is in general not operator specific.
```suggestion
        print("Storing test results in JUnit format")
```